external/dir.proj

```diff
@@ -13,13 +13,26 @@
     <Project Include="netfx/netfx.depproj" />
     <Project Include="runtime/runtime.depproj" />
     <Project Include="tools/tools.depproj" />
-    <Project Include="test-runtime/XUnit.Runtime.depproj"  Condition="'$(DotNetBuildFromSource)' != 'true'" />
     <Project Include="harvestPackages/harvestPackages.depproj" />
     <Project Include="binplacePackages/binplacePackages.depproj" />
-    <Project Include="docs/docs.depproj" Condition="'$(DotNetBuildFromSource)' != 'true'" />
-    <Project Include="optimizationData/optimizationData.depproj" Condition="'$(EnableNgenOptimization)' == 'true' AND '$(DotNetBuildFromSource)' != 'true'" />
-    <Project Condition="'$(ILLinkTrimAssembly)' != 'false'" Include="ILLink/ILLink.depproj" />
   </ItemGroup>
 
-  <Import Project="$(RepositoryEngineeringDir)dir.traversal.targets" />
+  <Target Name="IncludeProjectReferences" DependsOnTargets="FilterProjects">
```

Comment:
What's the purpose of this?

Reply:
This eliminates the need of the dir.traversal.targets and instead uses the SDK restore target.

Reply:
I see now.  That deserves a comment.